README.md

```diff
@@ -291,49 +291,14 @@ Default: `size of the folder`
 
 Estimate of the total amount of disk space required to install the named package, used in the [`Installed-Size` field of the `control` specification](https://www.debian.org/doc/debian-policy/ch-controlfields.html#s-f-Installed-Size).
 
-#### options.depends
+#### options.depends, recommends, suggests, enhances, preDepends
 Type: `Array[String]`
-Default: `[TrashDependsVersion, 'libgconf2-4', 'libgtk2.0-0', 'libnotify4', 'libnss3', 'libxtst6', 'xdg-utils']`
-Relationships to other packages, used in the [`Depends` field of the `control` specification](https://www.debian.org/doc/debian-policy/#binary-dependencies-depends-recommends-suggests-enhances-pre-depends).
+Default: see [`src/installer.js`](https://github.com/unindented/electron-installer-debian/blob/master/src/installer.js#L181)
```

Comment:
Linking to master is not the best, because the line number will change over time.

Reply:
that's true, what do you propose instead?

Reply:
I could use a Github permalink to that line, so even if it moves, the link will point to that particular commit and line. If changes are introduced that completely change how we handle defaults, then we could update the permalink, but that's less likely. Even if the line moves, the defaults won't really change. Once this PR is merged, I will push a new commit with the permalink to the most up-to-date commit if we decide to use this idea.
Also, I want to merge `getTrashDepends` and `getDefaultDepends` into one function that will only return the trash dependencies string. This way, we can keep the defaults under `getDefaultDepends` inside the `defaults` object which will be a lot easier to read.